src/server.js

```diff
@@ -7,7 +7,7 @@ fastify.register(require('fastify-cors'))
 fastify.register(require('fastify-postgres'), {
   connectionString: process.env.DATABASE_URL,
 })
-fastify.register(require('./routes'))
+fastify.register(require('./routes/userRoutes/userRoutes'))
 
 const start = async () => {
```

Comment:
A good practice I've found is to have function declarations in the global scope (i.e. async function() {}). This can help alleviate issues with "this". We will still keep arrow functions for everything else.